modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -586,6 +586,20 @@ public static Preferences getPreferences() {
     public sealed interface Preferences extends ObservableMap<String, Object>
             permits com.sun.javafx.application.preferences.PlatformPreferences {
 
+        /**
+         * Specifies whether applications should always show scroll bars. If not set, an application may
+         * choose to hide scroll bars that are not actively used, or make them smaller or less noticeable.
+         * <p>
+         * If the platform does not report this preference, this property defaults to {@code false}.
+         *
+         * @return the {@code persistentScrollBars} property
+         * @defaultValue {@code false}
+         * @since 24
+         */
+        ReadOnlyBooleanProperty persistentScrollBarsProperty();
```

Comment:
Do you also plan to add new platform-specific preferences or do you think this is unnecessary?

Reply:
I added the respective platform-specific keys to the `Preferences` documentation.